.travis.yml

```diff
@@ -22,26 +22,34 @@ before_install:
   - docker exec $CONTAINER dnf -y install libasan libubsan
   - docker exec $CONTAINER dnf -y install clang-analyzer
   - docker exec $CONTAINER dnf -y install mingw64-gcc mingw64-libffi mingw64-libtasn1 wine
+  - docker exec $CONTAINER dnf -y install opensc openssl
```

Comment:
why not all installations are done at the same time? wouldn't it make them faster?

Reply:
That's a good point; I arranged the `dnf install` invocation in https://github.com/p11-glue/p11-kit/pull/142/commits/05c257e5e71c78599cc6970c7a7438fd79bb863c.